src/main/java/com/xiaomi/infra/pegasus/client/PegasusClientInterface.java

```diff
@@ -3,6 +3,7 @@
 // can be found in the LICENSE file in the root directory of this source tree.
 package com.xiaomi.infra.pegasus.client;
 
+import com.xiaomi.infra.pegasus.rpc.TableOptions;
```

Comment:
You cannot expose the `rpc` module to our users.

`rpc.TableOptions` and `client.TableOptions` should be separated into different classes. Because the former could have internal data that should be obscure to our users.


I'd suggest this way:

```
package com.xiaomi.infra.pegasus.rpc;

import com.xiaomi.infra.pegasus.client.TableOptions;

class InternalTableOptions extends TableOptions {
```

and 

```
package com.xiaomi.infra.pegasus.client;

class TableOptions {
```

Reply:
When users using `TableOptions` to opentTable, for the `TableOptions` has no `KeyHasher`, How can  `cluster.openTable(tableName, tableOptions)`  be succeed？

So if don't want expose the `KeyHasher` to users, it should be move it out and as a dependency variable to init `openTable`:

```
cluster.openTable(tableName, KeyHasher, tableOptions)
```